src/app/pages/doc/overview.twig

```diff
@@ -46,9 +45,20 @@
             <ul class="actions">
               {% for docLink in docLinksLE %}
                 <li>
-                  <a href="{{ docLink.url }}" class="button special min-width">
-                    <i class="si si-book"></i> {{ docLink.displayText }}
-                  </a>
+                  <span class="button special min-width">
+                    <a href="{{ docLink.url }}" >
+                      <i class="si si-book"></i> {{ docLink.displayText }}
+                    </a>
+                    <span class="doc-dropdown" ontouchstart="$(this).toggleClass('active-dropdown');">
```

Comment:
I would add a method to the DocProvider which is accessible over RelaseInfo to evaluate this links.